config/cmake/jrunTest.cmake

```diff
@@ -49,7 +49,7 @@ endif ()
 
 message (STATUS "COMMAND: ${TEST_TESTER} -Xmx1024M -Dorg.slf4j.simpleLogger.defaultLog=${LOG_LEVEL} -Djava.library.path=\"${TEST_LIBRARY_DIRECTORY}\" -cp \"${TEST_CLASSPATH}\" ${TEST_ARGS} ${TEST_PROGRAM} ${ARGN}")
 
-if (WIN32 OR MINGW)
+if (WIN32)
```

Comment:
Another check needed - if this is on a linux host is the ENV{PATH} valid?

Reply:
yes, it will be used by wine